docs/Config.md

```diff
@@ -242,6 +242,24 @@ os:
 
 Lazygit will log an error if none of these options are set.
 
+You can give the number of lines you are currently pointing to in the line-by-line panel.
```

Comment:
What about: "You can specify a line number you are currently at when in the line-by-line mode."

Reply:
Thanks!